comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -187,6 +188,26 @@ class TcpTransportLayer(port: Int, cert: String, key: String, maxMessageSize: In
                })
     } yield result
 
+  private def deleteFile(path: Path): Task[Unit] = {
+    def delete(): Task[Unit] =
+      for {
+        result <- Task.delay(Try(path.toFile.delete).toEither)
```

Comment:
Why not `Task.delay(path.toFil.delete).attempt` ?

Reply:
yes, absolutely!